### PR TITLE
About page

### PR DESCRIPTION
**Pull request by boothy060590**

### CODE REVIEW

File: UI/IntroHero/index.js [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
+      <form className={introStyles.intro} onSubmit={e => {
+         e.preventDefault()
+         // Implementation of this function is an exercise for the reader.
+         navigate(
+            "/game",
+            {
+               state: { name },
+            }
+         );
+      }}>
```
Comment (petemcfarlane) [inline UI/IntroHero/index.js]: Think you missed this in a merge conflict?

File: package.json [hunk 2]
```diff
@@ -19,14 +19,19 @@
...
     "howler": "^2.2.1",
     "lodash": "^4.17.20",
     "markdown-to-jsx": "^7.1.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+<<<<<<< HEAD
+    "react-helmet": "^6.1.0",
+=======
     "react-tooltip": "^4.2.11",
+>>>>>>> origin/main
```
Comment (petemcfarlane) [inline package.json]: This needs fixing

Comment (petemcfarlane) [reply package.json]: probably add them both and run `yarn` to generate the lock file again

File: src/pages/game.js [hunk 3]
```diff
@@ -142,27 +143,29 @@ export default function Game({data, location}) {
...
-               computerName={computerName}
-               usersTurnCard={usersTurnCard}
-               computersTurnCard={computersTurnCard}
-               winner={roundWinner}
-               selectedAttribute={selectedAttribute}
-               incrementTurnCount={incrementTurnCount}
-               slamJams={slamJams}
-            ></Result>
-         )}
+         <InfoIcon>
```
Comment (petemcfarlane) [inline src/pages/game.js]: I would  have just replaced the fragment if you have a wrapper, e.g. `<>` becomes `<InfoIcon>`

**Review by petemcfarlane — CHANGES_REQUESTED** [review]
Just a couple of merge conflicts it seems, see comments, but otherwise 👍

File: package.json (hunk 2 above)
Comment (boothy060590) [reply package.json]: whoops done :-)

File: src/pages/game.js (hunk 3 above)
Comment (boothy060590) [reply src/pages/game.js]: Good spot. Done :-)

File: UI/IntroHero/index.js (hunk 1 above)
Comment (boothy060590) [reply UI/IntroHero/index.js]: I think this is right isn't it as we dont have setPage anymore?

Comment (boothy060590) [reply UI/IntroHero/index.js]: fixed

**Pull request merged**